.github/workflows/locize-push-missing-keys.yml

```
@@ -0,0 +1,26 @@
+name: Push New Keys to locize
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)